ubuntu/debian/libgz-transport-log.install

```diff
@@ -1 +1,3 @@
 usr/lib/*/lib*-transport[0-9][0-9]-log.so.*
+usr/lib/ruby/*/cmdlog[0-9][0-9].rb
+usr/share/*/transportlog[0-9][0-9].yaml
```

Comment:
Should we put these in the CLI package instead? Or do we need a `log-cli` package?

Reply:
I'm going to leave this one like this and merge the PR to fix the libexec issue